cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -907,33 +893,34 @@ toLedgerProposedPPUpdates era =
   . Map.mapKeysMonotonic (\(GenesisKeyHash kh) -> kh)
   . Map.map (toLedgerPParamsUpdate era)
 
-
+-- TODO: Stop using partial function and switch to the Either String variant
```

Comment:
Will this be done as part of this PR?

Reply:
I suspect this will have much ramification in downstream uses, so I would suggest to do it in a follow up PR. However, if really necessary I could squeeze it in into this one

Reply:
Created a ticket for this: https://github.com/input-output-hk/cardano-node/issues/5092